index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I would kindly advice that you include[ Meta tags ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta)as shown below. Meta tags help you provide additional information about your webpage if you. This helps Google (and other search engines) know more about your webpage.  

```suggestion
<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>HTML5 Starter Template</title>
  <meta name="description" content="Starter Template">
  <meta name="author" content="Gregry Pike">
  <link rel="stylesheet" href="css/styles.css?v=1.0">
  </head>
<body>
```

Reply:
- [ ]  Good clear and polite comment offered as a suggestion 👏🏾